src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
 
     private String name;
```

Comment:
&#x1F34E; **Issue**
Why not make it final ?
Also, please check out other classes for the same issue.